packages/dara-components/dara/components/smart/code_editor/code_editor.py

```diff
@@ -30,5 +32,16 @@ class CodeEditor(StyledComponentInstance):
 
     script: NonDataVariable
 
+    language: Optional[
```

Comment:
We should have added, but since de don't have, would you might adding a quick example of how to use the component with the new language prop?